controllers/virtualmachinepublishrequest/virtualmachinepublishrequest_controller.go

```diff
@@ -900,15 +916,17 @@ func (r *Reconciler) ReconcileNormal(ctx *context.VirtualMachinePublishRequestCo
 		if deleted {
 			skipPatch = true
 		}
-		isDeleted = deleted
-		return ctrl.Result{RequeueAfter: requeueAfter}, err
+		return ctrl.Result{RequeueAfter: requeueAfter}, false, err
```

Comment:
Two questions here:
1) why do you remove `isDeleted = deleted`? Shouldn't this still be used when registering metrics in defer?
2) I don't think your overhead `skipPath = true` is applied as you return hard-coded false at the end.

Reply:
That line was removed by accident. Added back.